analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldNotHaveIdenticalImplementations.CSharp9.cs

```diff
@@ -1,18 +1,19 @@
 ﻿using System;
 
-void Method1()
+void Method1()// Secondary
+              // Secondary@-1
```

Comment:
Let's introduce [] notation here to differentiate between to issues at line 10 and 16?

Reply:
I'm not sure I understand.
Could you provide me with an example?

Reply:
You can take a look at issues markup in the file analyzers\tests\SonarAnalyzer.UnitTest\TestCases\CastShouldNotBeDuplicated.CSharp10.cs